burstphoto/ContentView.swift

```diff
@@ -200,41 +204,64 @@ struct ProcessingView: View {
 
 struct SettingsView: View {
     @ObservedObject var settings: AppSettings
-    let tile_sizes = [16, 32, 64]
-    let search_distances = ["Low", "Medium", "High"]
+    let tile_sizes = ["Small", "Medium", "Large"]
+    let search_distances = ["Small", "Medium", "Large"]
     let merging_algorithms = ["Fast", "Higher quality"]
+    let exposure_controls = ["Off", "Balanced", "Brighter"]
```

Comment:
These three options need better names (as does "exposure control") and a description of what they affect.

Reply:
A table with description of the 'Exposure control' options was added to this PR.